Library/Formula/aurora-scheduler.rb

```diff
@@ -0,0 +1,65 @@
+class AuroraScheduler < Formula
+  desc "Apache Aurora Scheduler Client"
+  homepage "https://aurora.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=aurora/0.9.0/apache-aurora-0.9.0.tar.gz"
+  sha256 "16040866f3a799226452b1541892eb80ed3c61f47c33f1ccb0687fb5cf82767c"
+
+  depends_on "gradle" => :build
+  depends_on "mesos"
+  depends_on :java => "1.8+"
+
+  def pour_bottle?
+    quiet_system("/usr/libexec/java_home --version 1.8 --failfast")
+  end
+
+  def install
+    ENV["PIP_DEFAULT_TIMEOUT"] = "60"
+    ENV["LC_ALL"] = "en_US.UTF-8"
+    ENV["CFLAGS"] = "-Qunused-arguments"
+    ENV["CPPFLAGS"] = "-Qunused-arguments"
+    File.write(".pantsversion", File.open(".pantsversion", &:read).gsub(/0\.0\.32/, "0.0.53"))
+    tools = <<-EOS.undent
+      # common rev for all org.scala-lang%* artifacts
+      SCALA_REV = '2.10.4'
+
+      jar_library(name = 'scala-compiler',
+                  jars = [
+                    jar(org = 'org.scala-lang', name = 'scala-compiler', rev = SCALA_REV),
+                  ])
+
+      jar_library(name = 'scala-library',
+                  jars = [
+                    jar(org = 'org.scala-lang', name = 'scala-library', rev = SCALA_REV),
+                  ])
+
+      jar_library(name = 'scala-repl',
+                  jars = [
+                    jar(org = 'org.scala-lang', name = 'jline', rev = SCALA_REV, intransitive = True),
+                  ],
+                  dependencies = [
+                    ':scala-compiler',
+                    ':scala-library',
+                  ])
+    EOS
+    File.open("BUILD.tools", "a+") { |f| f.puts(tools) }
```

Comment:
El Capitan : pants 0.0.53
work in https://github.com/pantsbuild/pants/blob/master/BUILD.tools